src/main/webapp/js/omniComment.js

```diff
@@ -1,15 +1,13 @@
 $(document).ready(function() {
-    var classNameForCommentsInFeedbackResponse = 'list-group-item list-group-item-warning giver_display-by';
-    var classNameForCommentsInStudentRecords = 'panel panel-info student-record-comments giver_display-by';
-    
     function isRedirectToSpecificComment() {
-        return $(location).attr('href').indexOf('#') !== -1;
+        return location.href.indexOf('#') !== -1;
```

Comment:
I remember including a polyfill of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes, try to use it if possible, it's semantically more meaningful than ```.indexOf...```